pyogrio/geopandas.py

```diff
@@ -223,8 +250,22 @@ def read_dataframe(
     arrow_to_pandas_kwargs : dict, optional (default: None)
         When `use_arrow` is True, these kwargs will be passed to the `to_pandas`_
         call for the arrow to pandas conversion.
+    datetimes : str, optional (default: "UTC")
+        The way datetime columns should be returned. Possible values:
+
+        - **"UTC"**: return all datetime columns as pandas datetime64 columns
+          converted to UTC. Naive datetimes (without timezone information) will
+          be assumed to be in UTC timezone.
+        - **"DATETIME"**: return datetimes in the timezone as they were read
+          from the data source. Columns with values in a single timezone or
+          without timezone information will be returned as pandas datetime64
+          columns. Columns with mixed timezone data are returned as object
```

Comment:
Small nitpick on the wording, but I think we should be clear about that it are "timezone-aware columns with mixed offsets", because the general case of mixed offsets (due to DST changes throughout the year) is still a single time _zone_. 
The issue is that GDAL does not actually support the concept of time zones, only offsets.

Reply:
True, I tried to clarify this.